src/server/scripts/BrokenIsles/zone_mardum.cpp

```diff
@@ -1645,6 +1653,334 @@ class spell_give_me_sight_beyond_sight_periodic : public AuraScript
     }
 };
 
+enum HiddenNoMoreData
+{
+    QUEST_HIDDEN_NO_MORE                = 39495,
+
+    NPC_JAYCE_CRYPTIC_HOLLOW            = 96436,
+    NPC_SHIVARRA_CRYPTIC_HOLLOW         = 96504,
+    NPC_ROCKSLIDE_KILLCREDIT            = 98755,
+
+    SAY_JAYCE_ATTACK_ILLIDARI           = 1,
+
+    ACTION_HIDDEN_NO_MORE_EYEBEAM       = 1,
+    ACTION_HIDDEN_NO_MORE_MOVE          = 2,
+
+    SPELL_VISUAL_KIT_SHIVARRA_TELEPORT  = 43182,
+    SPELL_VISUAL_KIT_SHIVARRA_TELEPORT2 = 43576,
+    SPELL_VISUAL_KIT_SHIVARRA_TELEPORT3 = 45431,
+
+    SPELL_ABANDON_HIDDEN_NO_MORE        = 194376,
+    SPELL_COSMETIC_EYE_BEAM_01_FEMALE   = 200754,
+    SPELL_COSMETIC_EYE_BEAM_01_MALE     = 194326
+};
+
+enum HiddenNoMorePaths
+{
+    POINT_JAYCE_DARKWEAVER_PREPARE_JUMP     = 1,
+    POINT_JAYCE_DARKWEAVER_JUMP_TO_CAVE     = 2,
+
+    PATH_JAYCE_DARKWEAVER_RUN_INTO_CAVE     = 9643600,
+
+    PATH_CRYPTIC_HOLLOW_DEMON_HUNTER_01     = 10178700,
+    PATH_CRYPTIC_HOLLOW_DEMON_HUNTER_02     = 10178701,
+    PATH_CRYPTIC_HOLLOW_DEMON_HUNTER_03     = 10178800,
+    PATH_CRYPTIC_HOLLOW_DEMON_HUNTER_04     = 10178900,
+    PATH_CRYPTIC_HOLLOW_DEMON_HUNTER_05     = 10179000,
+    PATH_CRYPTIC_HOLLOW_COILSKAR_01         = 9650200,
+    PATH_CRYPTIC_HOLLOW_COILSKAR_02         = 9650201,
+    PATH_CRYPTIC_HOLLOW_COILSKAR_03         = 9650300,
+    PATH_CRYPTIC_HOLLOW_ASHTONGUE_01        = 9650000,
+    PATH_CRYPTIC_HOLLOW_ASHTONGUE_02        = 9650001,
+    PATH_CRYPTIC_HOLLOW_ASHTONGUE_03        = 9650100
+};
+
+// 39495 - Hidden No More
+class quest_hidden_no_more : public QuestScript
+{
+public:
+    quest_hidden_no_more() : QuestScript("quest_hidden_no_more") { }
+
+    void OnQuestStatusChange(Player* player, Quest const* /*quest*/, QuestStatus /*oldStatus*/, QuestStatus newStatus) override
+    {
+        if (newStatus == QUEST_STATUS_NONE)
+        {
+            player->CastSpell(nullptr, SPELL_ABANDON_HIDDEN_NO_MORE, true);
+            PhasingHandler::OnConditionChange(player);
+        }
+    }
+};
+
+// 96436 - Jayce Darkweaver
+struct npc_jayce_darkweaver_cryptic_hollow : public ScriptedAI
+{
+    npc_jayce_darkweaver_cryptic_hollow(Creature* creature) : ScriptedAI(creature) { }
+
+    void OnQuestAccept(Player* player, Quest const* quest) override
+    {
+        if (quest->GetQuestId() == QUEST_HIDDEN_NO_MORE)
+        {
+            me->SummonPersonalClone(me->GetPosition(), TEMPSUMMON_MANUAL_DESPAWN, 0s, 0, 0, player);
+            PhasingHandler::OnConditionChange(player);
```

Comment:
we should be able to drop this after c90dbeb648f4bba2c354ed37b89f696f99e5bc94 as the quest has flags `0x00400000 (QUEST_FLAGS_UPDATE_PHASESHIFT)` set